README.md

```diff
@@ -245,6 +245,21 @@ julia> classify([0.1,0.2,0.6,0.1], LabelEnc.OneOfK) # single observation
 # 3
 ```
 
+
+`NativeLabels` maps between data of an arbitary type (e.g. Strings) and the other label types (Normally `LabelEnc.Indices{Int}` for an integer index).
+When using it, you should always save the encoding in a variable, and pass it as an argument to `convertlabel`;
+as otherwise the encoding will be inferred each time, so will normally encode differently for different inputs.
+
+```julia
+julia> enc = labelenc(["copper", "tin", "gold"])
```

Comment:
That works, but the purpose of `labelenc` is that one passes a vector of targets to it. I guess it doesn't matter much unless one assumes that the order of elements always matter.

for example `labelenc([1, 0])` and `labelenc([0, 1])` give the same result

Reply:
`labelenc(["gold", "copper"])` and `labelenc(["copper", "gold"])` give different results.

Sometimes the order matters.
For me, today, the order matter.
Because that Index is what I use to look up a word embedding.

(Last time I did this the order didn't matter so long as it was consistent. Because I was training the work embeddings)

Reply:
Basically though, 
I'm not sure what exactly you mean

Reply:
Right, well, my argument was more that the order doesn't always matter to the function. It should for non-numbers and non-bools. Its also not sensitive to multiple occurances.
i.e. `labelenc(["gold", "copper", "gold", "gold"])` just silently ignores multiple occurnaces. This is on purpose since the parameters is assumed to be an array of observations.

I'd say the cleanest way to specify a `NativeLabels` encoding is using its constructor
`LabelEnc.NativeLabels(["gold", "copper"])`